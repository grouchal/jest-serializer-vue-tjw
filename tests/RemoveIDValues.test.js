const helpers = require('./helpers.js');

describe('RemoveIDValues', () => {
  const commented = `
<div>
  <span id="123">3</span>
  <div>
    <div id="234">
  </div>
</div>
`;

  test('ID values removed', () => {
    helpers.mockSettings({
      removeIdValues: true
    });

    expect(commented.trim())
      .toMatchSnapshot();
  });

  test('ID values remain', () => {
    helpers.mockSettings({
      removeIdValues: false
    });

    expect(commented.trim())
      .toMatchSnapshot();
  });
});
