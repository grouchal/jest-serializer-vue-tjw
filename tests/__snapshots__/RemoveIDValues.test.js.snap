// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RemoveIDValues ID values remain 1`] = `
<div>
  <span id="123">3</span>
  <div>
    <div id="234">
    </div>
  </div>
</div>
`;

exports[`RemoveIDValues ID values removed 1`] = `
<div>
  <span id>3</span>
  <div>
    <div id>
    </div>
  </div>
</div>
`;
